tests_requirements.txt

```diff
@@ -16,4 +16,5 @@ azure-mgmt-billing==6.0.0
 # EC2 LongRun Required by Jira
 aiohttp==3.8.1
 urllib3==1.26.7
-oauthlib~=3.1.1
\ No newline at end of file
+oauthlib~=3.1.1
+pytest
```

Comment:
Add it in Alphabetical place after pandas.
Do we still need the "EC2 LongRun Required by Jira" packages here and in requirements ?

Reply:
ok.
Yes, we need it. This package will help to connect JiraAPI